### PR TITLE
chore(cspi): add zpool status type

### PR DESCRIPTION
**Pull request by sonasingh46**
This PR adds the zpool status types.
Please reffer following OEP for more details : 
https://github.com/openebs/openebs/pull/2942/files
Signed-off-by: Ashutosh Kumar <ashutosh.kumar@mayadata.io>

### CODE REVIEW

**Review by prateekpandey14 — APPROVED** [review]
lgtm, minor comment

File: pkg/apis/cstor/v1/cstorpoolinstance.go [hunk 1]
```diff
@@ -68,17 +68,17 @@ const (
 	// CStorPoolStatusEmpty ensures the create operation is to be done, if import fails.
 	CStorPoolStatusEmpty CStorPoolInstancePhase = ""
 	// CStorPoolStatusOnline signifies that the pool is online.
-	CStorPoolStatusOnline CStorPoolInstancePhase = "Healthy"
+	CStorPoolStatusOnline CStorPoolInstancePhase = "ONLINE"
```
Comment (prateekpandey14) [inline pkg/apis/cstor/v1/cstorpoolinstance.go:71]: we have to update the `internalapis/apis` as well

**Review by mittachaitu — APPROVED** [review]
We need to make changes in internal APIs also

File: pkg/apis/cstor/v1/cstorpoolinstance.go (hunk 1 above)
Comment (sonasingh46) [reply pkg/apis/cstor/v1/cstorpoolinstance.go:71]: Well -- these constants are not present/required in the internal apis.

Comment (sonasingh46) [reply pkg/apis/cstor/v1/cstorpoolinstance.go:71]: For consistency, will push that -- and later we can decide whether we need those in internal apis or not.

**Pull request merged**